pages/studentInstanceQuestionHomework/studentInstanceQuestionHomework.js

```diff
@@ -32,10 +32,19 @@ function processSubmission(req, res, callback) {
         submitted_answer: submitted_answer,
         credit: res.locals.authz_result.credit,
         mode: res.locals.authz_data.mode,
+        assessment_instance_id: res.locals.assessment_instance.id,
+        assessment_id: res.locals.assessment.id,
+        instance_question_id: res.locals.instance_question.id,
     };
     sqldb.callOneRow('variants_ensure_instance_question', [submission.variant_id, res.locals.instance_question.id], (err, result) => {
         if (ERR(err, callback)) return;
         const variant = result.rows[0];
+        // There is going to be a situation where the file is needed for grading (likely!) so we would
+        // have to go inside the save and grade submission logic to let it in but not save it.
```

Comment:
In the first callback before a submission is saved, we remove the file so it does not enter the database. We can cache the data in a variable here and put it back in the grader for later.

OR, we can start parsing out the file data in the SQL. Any preference? It seems like a much easier problem to deal with at this scope. 

We also, likely, have to submit this data for External Grading. I have to start analyzing that logic to understand our options.